src/components/App/container/App.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+import { getUserDetails } from '@/reducers/user';
+import App from '../layout/App';
+
+const mapStateToProps = state => ({
```

Comment:
App-component does not uses these props. You can replace them by null, and win some optimisation score.